vtex-apps/order-placed
# Feat/add css handles

#trivial 
#### What is the purpose of this pull request?

Add css handles to ease the layout customization.

#### How should this be manually tested?

1) Go to https://kiwi--storecomponents.myvtex.com/checkout/orderPlaced/?og=984210525286
2) check the markup for the css classes

#### Types of changes
